Source/Storage/Storage+Channel.swift

```diff
@@ -135,7 +130,7 @@ extension Storage {
         }
 
         public func getCard() throws -> Card {
-            guard self.type == .single, let card = self.cards.first else {
+            guard (self.type == .single) || (self.type == .singleRatchet), let card = self.cards.first else {
```

Comment:
Let's use switch here for channel.type. Otherwise, we'll miss this place someday